doc/common/chapter00.md

```diff
@@ -208,49 +210,49 @@ The CNTT work is not done in a vacuum.  The intention from the beginning was to
-<p align="center"><b>Figure 6:</b> CNTT Relationship with OPNFV and OVP.</p>
+<p align="center"><img src="./figures/tech_relation.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 6:</b> Anuket Relationship with OVP.</p>
 
 As can be seen from the above figure, roles and responsibilities are as follows:
-- CNTT specifies lab requirements in the **Reference Implementation** document which will be used by **OPNFV** to define what labs can be used within the community for the purpose of installing and testing CNTT conformant cloud infrastructure implementations.
-- CNTT includes a lab Playbook in its **Reference Implementation** detailing available suitable labs to run and test cloud infrastructure implementations with access details and processes.
-- CNTT specifies installer requirements in the **Reference Implementation** document will be used by **OPNFV** to determine a suitable installer to be used to install a Reference Implementation of cloud infrastructure that are conformant to CNTT specifications.
-- CNTT includes an installation Playbook in it's **Reference Implementation** detailing instructions of how to install an OPNFV reference implementation using community installers. The **Reference Implementation** created by **OPNFV** is expected to be fully conformant to CNTT specifications and must pass all test suites defined in the CNTT **Reference Conformance** document.
-- CNTT specifies testing framework requirements in the **Reference Conformance** document will be used by OPNFV to determine a suitable testing framework and portals to be used for the purpose of running test suites and tools, and carry out badging processes.
-- CNTT defines high level test cases in the **Reference Conformance** document that relates to requirements coming from both the **Reference Model** and **Reference Architecture** to be used by **OPNFV** to determine what testing projects within the community are suitable to deliver those tests and convert them into low level test cases that can be implemented within those **OPNFV** testing Projects.
-- CNTT includes a traceability matrix in its **Reference Conformance** document detailing every test case (or group of test cases) available in the community and map them to the high level test case definition and the requirements they are fulfilling.
-- CNTT includes a testing Playbook in its **Reference Conformance** document detailing instructions of how to run **OPNFV** testing framework and test cases against commercial NFV products (infrastructure and workload) to check conformance to CNTT specifications. CNTT testing Playbook will also details instructions of how to submit testing results for the **OVP** badging process.
+- Anuket specifies lab requirements in the **Reference Implementation** document which will be used to define what labs can be used within the community for the purpose of installing and testing Anuket conformant cloud infrastructure implementations.
```

Comment:
```suggestion
- Anuket specifies lab requirements in the **Reference Implementation** document that will be used to define what labs can be used within the community for the purpose of installing and testing cloud infrastructure implementations for conformance with Anuket specifications.
```

Reply:
Could you please clarify whether the lab requirements are related to testing the Anuket conformance of the implementations or the implementations that are Anuket-compliant?

Reply:
@lylavoie Can you help?